src/main/java/mzaba/avroprotocol/kafka/controller/SpringAvroRestController.java

```diff
@@ -0,0 +1,27 @@
+package mzaba.avroprotocol.kafka.controller;
+
+import lombok.RequiredArgsConstructor;
+import mzaba.avroprotocol.kafka.model.BookModel;
+import mzaba.avroprotocol.kafka.producer.SpringAvroProducer;
+import mzaba.avroprotocol.schema.Book;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Random;
+
+@RestController
+@RequiredArgsConstructor
+public class SpringAvroRestController {
+
+    private SpringAvroProducer springAvroProducer;
+
+    @PostMapping(value = "/sendBook")
```

Comment:
Generally, using verbs in an endpoint is not a very good idea according to REST API.
The default approach to creating a resource will be **POST /books** with **201** response code